docs/ref/contrib/postgres/window.txt

```diff
@@ -0,0 +1,253 @@
+================
+Window functions
+================
+
+.. module:: django.contrib.postgres.window
+    :synopsis: Window expression API.
+
+    .. versionadded:: 2.0
```

Comment:
Is this intended to be added on 1.11 or version 2.0? found the change log in 1.11 though